years_to_hours.rb

```diff
@@ -0,0 +1,23 @@
+Ruby Program
+
+def years_to_hours
+  puts "Enter a number of years"
+  years = gets.chomp # this returns a string
+  years = years.to_i # this converts a string to an integer
+  hours = years * 365 * 24
+  puts "That's #{hours} hours."
+  
+  puts "Enter a number of decades"
+  decades = gets.chomp # this returns a string
+  decades = decades.to_i # this converts a string to an integer
+  mins = decades * 10 * 365 * 24 * 60
+  puts "That's #{mins} minutes."
+  
+  puts "Enter your age"
+  yearsold = gets.chomp # this returns a string
+  age = yearsold.to_i # this converts a string to an integer
```

Comment:
As a matter of style, we generally use underscores ("[snake case](https://en.wikipedia.org/wiki/Snake_case)") in Ruby to separate words in a variable.

```suggestion
  years_old = gets.chomp # this returns a string
  age = years_old.to_i # this converts a string to an integer
```